Mathlib/Dynamics/OmegaLimit.lean

```diff
@@ -227,7 +227,7 @@ theorem eventually_closure_subset_of_isCompact_absorbing_of_isOpen_of_omegaLimit
     apply Subset.trans hg₃
     simp only [j, iUnion_subset_iff, compl_subset_compl]
     intros u hu
-    mono
+    unfold w; gcongr
```

Comment:
How would you feel about putting these on separate lines?

Reply:
Equally fine, so I'm happy to do so.
```suggestion
    unfold w
    gcongr